.github/workflows/performance.yml

```diff
@@ -0,0 +1,224 @@
+  # - Install WordPress Importer plugin.
+  # - Import mock data.
+  # - Update permalink structure.
+  # - Run the performance tests (current commit).
+  # - Print performance tests results.
+  # - Set the environment to the baseline version.
+  # - Run baseline performance tests.
+  # - Print base line performance tests results.
+  # - Set the base sha.
+  # - Set commit details.
+  # - Publish performance results.
+  # - Ensures version-controlled files are not modified or deleted.
+  performance:
+    name: Run performance tests
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
+
+    steps:
+      - name: Configure environment variables
+        run: |
+          echo "PHP_FPM_UID=$(id -u)" >> $GITHUB_ENV
+          echo "PHP_FPM_GID=$(id -g)" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Log debug information
+        run: |
+          npm --version
+          node --version
+          curl --version
+          git --version
+          svn --version
+          locale -a
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Build WordPress
+        run: npm run build
+
+      - name: Start Docker environment
+        run: |
+          npm run env:start
+
+      - name: Log running Docker containers
+        run: docker ps -a
+
+      - name: Docker debug information
+        run: |
+          docker -v
+          docker-compose -v
+          docker-compose run --rm mysql mysql --version
+          docker-compose run --rm php php --version
+          docker-compose run --rm php php -m
+          docker-compose run --rm php php -i
+          docker-compose run --rm php locale -a
+
+      - name: Install WordPress
+        run: npm run env:install
+
+      - name: Install WordPress Importer plugin
+        run: npm run env:cli -- plugin install wordpress-importer --activate ${{ env.BUILD_PATH }}
+
+      - name: Import mock data
+        run: |
+          curl -O https://raw.githubusercontent.com/WPTT/theme-test-data/b9752e0533a5acbb876951a8cbb5bcc69a56474c/themeunittestdata.wordpress.xml
+          npm run env:cli -- import themeunittestdata.wordpress.xml --authors=create ${{ env.BUILD_PATH }}
+          rm themeunittestdata.wordpress.xml
+
+      - name: Update permalink structure
+        run: |
+          npm run env:cli -- rewrite structure '/%year%/%monthnum%/%postname%/' ${{ env.BUILD_PATH }}
+
+      - name: Install MU plugin
+        run: |
+          mkdir ./build/wp-content/mu-plugins
+          cp ./tests/performance/mu-plugins/server-timing.php ./build/wp-content/mu-plugins/server-timing.php
```

Comment:
We should change the hard-coded `build` directory to reference the `LOCAL_DIR` env variable instead.

Reply:
Also handled in [2e15306](https://github.com/WordPress/wordpress-develop/pull/4139/commits/2e153061a54d7fbbb46cf7b356a18285d61f2931)